### PR TITLE
Bug 1860985: Updated minimum requirement for Internal And Internal-Attached Devices

### PR DESCRIPTION
**Pull request by gnehapk**
- Updated minimum requirement text for internal and attached devices mode
- Updated text and conditions for Minimal cluster deployment Alert for internal and attached devices mode. For both, it should be same.

[Ignore the error message as this is just for making the changes visible)
Screenshot from 2020-09-25 17-08-22

Screenshot from 2020-09-25 17-08-10

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/utils/common-ocs-install-el.tsx [hunk 1]
```diff
@@ -22,9 +23,7 @@ type MinimalDeploymentAlertProps = {
   isInternalMode?: boolean;
 };
 
-export const MinimalDeploymentAlert: React.FC<MinimalDeploymentAlertProps> = ({
-  isInternalMode,
-}) => (
+export const MinimalDeploymentAlert: React.FC<MinimalDeploymentAlertProps> = () => (
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/utils/common-ocs-install-el.tsx:26]:
```suggestion
export const MinimalDeploymentAlert: React.FC = () => (
```
Lets remove the type declaration as well.

**Review by bipuladh — COMMENTED** [review]
nit

File: frontend/packages/ceph-storage-plugin/src/utils/common-ocs-install-el.tsx (hunk 1 above)
Comment (gnehapk) [reply frontend/packages/ceph-storage-plugin/src/utils/common-ocs-install-el.tsx:26]: ack. Missed it

**Review by bipuladh — APPROVED** [review]
/lgtm

**Comment by gnehapk** [comment]
/hold
Waiting for confirmation for Alert text

**Review by bipuladh — APPROVED** [review]
/lgtm

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx [hunk 2]
```diff
@@ -100,7 +100,7 @@ export const CreateOCS = withHandlePromise<CreateOCSProps & HandlePromiseProps>(
   const isMinimalSupported = useFlag(OCS_SUPPORT_FLAGS.MINIMAL_DEPLOYMENT);
   const isEncryptionSupported = useFlag(OCS_SUPPORT_FLAGS.ENCRPYTION);
 
-  const isMinimal = shouldDeployAttachedAsMinimal(nodes);
+  const isMinimal = shouldDeployAsMinimal(nodes);
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx:103]:
```suggestion
  const isMinimal = nodes.length >= 3 ? shouldDeployAsMinimal(nodes) : false;
```

**Review by bipuladh — CHANGES_REQUESTED** [review]
For LSO this is required because we shouldn't be showing the error and warning at the same time.
/lgtm cancel

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**